src/main/resources/lang/en_uk.yml

```diff
@@ -53,6 +54,27 @@ statuses:
     error_loading: "Error loading cat picture! {frowning_emoji}"
   dog:
     error_loading: "Error loading dog picture! {frowning_emoji}"
+  # Developer
+  dump:
+    not_enough_arguments: "Hmmm either pick: `threads`, `commands`, `permissions` or `guild`"
+    command: "Command"
+    module: "Module"
+    permission: "Permission"
+    subcommand: "Subcommand"
+    discord_perms: "Discord permissions"
+    default_permission: "Default permission"
+    no_permission: "No permission"
+    cannot_find_argument: "I can't seem to find that argument {thinking_emoji}"
```

Comment:
The  ones above are pretty generic, leave them for now but we may need to move them in future

Reply:
Don't listen to that, add line 60-65 to `words` section